Task2/controllers/users.js

```diff
@@ -0,0 +1,89 @@
+const usersData = require('./../data/users.json');
+import { ErrorMessages } from './../config/constants';
+import { v4 as uuidv4 } from 'uuid';
+import { omit } from 'lodash';
+
+const getUserById = (req, res) => {
```

Comment:
Service isn't supposed to get `(req, res)`. It should only be responsible for processing data. `(req, res)` should be handled in routes.

Reply:
Not applicable